server/api/api.go

```diff
@@ -114,12 +118,85 @@ func (h *Handler) setLink(w http.ResponseWriter, r *http.Request) {
 	status := http.StatusNotModified
 	if changed {
 		if err := h.store.SaveLinks(links); err != nil {
-			h.handleError(w, errors.Wrap(err, "unable to save link"))
+			h.handleError(w, errors.Wrap(err, "unable to save the link"))
 			return
 		}
 		status = http.StatusOK
 	}
 
+	ReturnStatusOK(status, w)
+}
+
+func (h *Handler) deleteLink(w http.ResponseWriter, r *http.Request) {
+	autolinkName := r.URL.Query().Get(autolinkclient.AutolinkNameQueryParam)
+	if autolinkName == "" {
+		h.handleError(w, errors.New("autolink name or pattern should not be empty"))
+		return
+	}
+
+	links := h.store.GetLinks()
+	found := false
+	for i := 0; i < len(links); i++ {
+		if links[i].Name == autolinkName || links[i].Pattern == autolinkName {
```

Comment:
I wonder if we should only match by `link.Name` to simplify both the API and the endpoint code, as `link.Name` should be unique and there are no chances of the same parameter matching more than one link

Reply:
sure @mgdelacroix Doing this change.